src/Microsoft.VisualStudio.Editors/DesignerFramework/ToolStripImageComboBox.vb

```diff
@@ -8,11 +8,11 @@ Namespace Microsoft.VisualStudio.Editors.DesignerFramework
 
 #If False Then 'Not currently needed
 
-    '@ <summary>
-    '@ This class is a toolstrip item that contains a combobox which is capable
-    '@   of holding images and having a different font for each entry in it.
-    '@ </summary>
-    '@ <remarks></remarks>
+    ''' <summary>
+    ''' This class is a toolstrip item that contains a combobox which is capable
+    '''   of holding images and having a different font for each entry in it.
+    ''' </summary>
+    ''' <remarks></remarks>
```

Comment:
:bulb: Empty `<remarks>` element can be removed with no negative impact (occurs a few times in this PR)

Reply:
I'm going to do this in a follow up PR, these two projects have massive swab of empty XML elements and it's a giant rabbit hole.